vsutil.py

```diff
@@ -1,9 +1,10 @@
 """
 VSUtil. A collection of general-purpose VapourSynth functions to be reused in modules and scripts.
 """
-__all__ = ['core', 'fallback', 'frame2clip', 'get_depth', 'get_plane_size', 'get_subsampling', 'get_w', 'get_y',
-           'insert_clip', 'is_image', 'iterate', 'join', 'plane', 'split', 'vs']
+__all__ = ['Dither', 'Range', 'core', 'depth', 'fallback', 'frame2clip', 'get_depth', 'get_plane_size',
```

Comment:
Why do you export `core`? Actually, why did we ever export `core`?

Reply:
FWIW, `vs` is also exported.

Reply:
From the original commit, <https://github.com/Irrational-Encoding-Wizardry/vsutil/pull/28/commits/45d22b76caa03bd3b785fe22a0cbd611ceb79246>
> This allows cleaner looking scripts, and might as well be the
recommended way of import from vsutil _in scripts_ not other modules.
With `core` and `vs` being defined in __all__, a VS script will not need
both of the standard VS lines if using the wild card import.

In someone's encoding script, a `from vsutil import *` will allow them to access all `vs` members and use the normal `core.Etc.etc` syntax without having to write:

```py
import vapoursynth as vs
from vsutil import *
core = vs.core
```
at the top of every script.

Reply:
I think I will move this into a separate issue. No need to actually talk about here.